ouroboros-consensus-cardano/src/unstable-cardano-testlib/Test/Consensus/Cardano/ProtocolInfo.hs

```diff
@@ -326,6 +326,7 @@ mkTestProtocolInfo
               SL.exampleAlonzoGenesis
               SL.exampleConwayGenesis
           )
+          mempty
```

Comment:
I think it makes sense to also use `emptyCheckpointsMap` here as you did in most other places :+1:

Reply:
Adopted.